qiskit/aqua/algorithms/amplitude_estimators/iqae.py

```diff
@@ -60,10 +75,20 @@ def __init__(self, epsilon: float, alpha: float,
                 each iteration, can be 'chernoff' for the Chernoff intervals or 'beta' for the
                 Clopper-Pearson intervals (default)
             min_ratio: Minimal q-ratio (K_{i+1} / K_i) for FindNextK
```

Comment:
Should "(K_{i+1} / K_i)"  be  :math:`(K_{i+1} / K_i)` ?

Reply:
yep, done in e1b0490 👍